airflow/operators/latest_only_operator.py

```diff
@@ -49,38 +49,40 @@ def execute(self, context):
 
         if not left_window < now <= right_window:
             logging.info('Not latest execution, skipping downstream.')
-            session = settings.Session()
+            downstream_task_ids = context['task'].downstream_task_ids
+            if downstream_task_ids:
```

Comment:
Here we create DB session only if we have task ids to process, otherwise it still gives empty sequence, so doesn't make sense to do DB request.